stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/Constants.java

```diff
@@ -180,6 +183,9 @@
     public static final int SSL_ENABLED_PROTOCOLS_INDEX = 4;
     public static final int CIPHERS_INDEX = 5;
     public static final int SSL_PROTOCOL_INDEX = 6;
+    public static final int CERTIFICATE_REVOCATION_VERIFIER_INDEX = 0;
+    public static final int CACHE_SIZE = 0;
+    public static final int CACHE_DELAY = 1;
```

Comment:
These 2 are also indices right? If so, let's change this to conform to the format followed by other indices.